components/coachindicator/stories/template.js

```diff
@@ -25,3 +26,44 @@ export const Template = ({
 		`)}
 	</div>
 `;
+
+/* This template group showcases multiple coach indicator variants at once. */
+export const AllVariantsCoachIndicatorGroup = (args, context) => Container({
+	withBorder: false,
+	content: html`
+		${Container({
+			direction: "column",
+			withBorder: false,
+			heading: "Default",
+			content: html`
+				${Template({
+					...args,
+					variant: "default"
+				}, context)}
+			`
```

Comment:
```suggestion
			content: Template({ ...args, variant: "default" }, context)
```

Another simplification opportunity here too